src/app/settings/views/Configuration/Security/Security.test.tsx

```diff
@@ -7,16 +7,23 @@ import {
   rootState as rootStateFactory,
   tlsCertificate as tlsCertificateFactory,
   tlsCertificateState as tlsCertificateStateFactory,
+  controller as controllerFactory,
+  controllerState as controllerStateFactory,
+  vaultEnabledState as vaultEnabledStateFactory,
 } from "testing/factories";
 import { renderWithBrowserRouter } from "testing/utils";
 
-it("displays loading text if TLS certificate has not loaded", () => {
+it("displays loading text if TLS certificate or Vault Status has not loaded", () => {
   const state = rootStateFactory({
     general: generalStateFactory({
       tlsCertificate: tlsCertificateStateFactory({
         data: null,
         loaded: false,
       }),
+      vaultEnabled: vaultEnabledStateFactory({
+        data: false,
```

Comment:
Isn't the data initially set to `null`?

Reply:
It's initially set to false in the selector since it's a boolean value, I can change it to accept null though?